packages/forma-36-website/src/content/getting-started.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: 'Getting started'
+---
+
+## Installation
+
+### Install components
+
+Inside your React project directory, install Forma 36 by running either of the following:
+
+**When using NPM:**
+
+```bash static=true
+npm i @contentful/f36-components
+```
+
+**When using Yarn:**
+
+```bash static=true
+yarn add @contentful/f36-components`
+```
+
+When you install `@contentful/f36-components` package, tree shaking will make your bundle contain only components that you actually import and need.
+
+#### Installing components as separate packages
+
+Although we recommend using `@contentful/f36-components`, you can install Forma 36 React components as separate packages if that's what you need for your project. For example, here's how to install just the button:
+
+**When using NPM:**
+
+```bash static=true
+npm i @contentful/f36-button
+```
+
+**When using Yarn:**
+
+```bash static=true
+yarn add @contentful/f36-button`
+```
+
+### Install icons
+
+Forma 36 icons are a separate package to reduce the size of the main bundle. If you need to use icons in your project, install them separatly by running either of the following:
+
+**When using NPM:**
+
+```bash static=true
+npm i @contentful/f36-icons
+```
+
+**When using Yarn:**
+
+```bash static=true
+yarn add @contentful/f36-icons`
+```
+
+#### Additional learning
+
+Forma 36 components are build using [TypeScript](https://www.typescriptlang.org/) and [React](https://reactjs.org/) 📚
```

Comment:
We should add a recommendation in this section, otherwise we are just making a statement, without any call for action. Something like:

For more information, see the [TypeScript](https://www.typescriptlang.org/) and [React](https://reactjs.org/) official documentation.